std_msgs/Header header
cube_tracker/Tracker[] trackers